beta-3-setup.md

```diff
@@ -3762,6 +3762,10 @@ Running ansible:
     export ROUTE_53_WILDCARD_ZONE=cloudapps.example.com
     export ROUTE_53_HOST_ZONE=example.com
 
+### Bootstrap the playbook directories
+
+    ansible-playbook clone_and_setup_repo.yml
```

Comment:
On 22/05/15 00:20 -0700, Juergen Hoffmann wrote:
>> +    ansible-playbook clone_and_setup_repo.yml
>
>@detiber Hi Jason, this should probably read `ansible-playbook -i 'localhost,' clone_and_setup_repo.yml`

That should not be needed.  localhost is treated specially in ansible and does 
not need to be listed in any inventory.  The default ansible hosts file should 
be good enough to kick off the playbook (unless there is no /etc/ansible/hosts 
file, in which case, an inventory of some type would need to be specified).